ci/credentials.yml.example

```diff
@@ -0,0 +1,13 @@
+concourse-slack-webhook-url: https://hooks.slack.com/services/xxx
+concourse-slack-channel: '#concourse'
```

Comment:
NBD, but you're not using these other fields in your pipeline, you just need the webhook-url